### DISCUSSION TITLE
Shell launching with rv.exe

### DISCUSSION THREAD

**Discussion by dkeraudren** [AcademySoftwareFoundation/OpenRV#247] [General]
When launching open rv, a shell window is launched along it ?
Is there a way to prevent that from happening ?

**Comment by richardssam** [comment]
I'm seeing the same thing. Do we need to make a wrapper that launches itself in a background mode?

**Comment by BrianHanke** [comment]
It would be nice to have no shell on Windows. It's not really the "Windows way." It just means you have an extra taskbar icon and alt-tab entry taking up space. If you need info about errors and such you can open the console within RV.

**Comment by richardssam** [comment]
Made a PR #266 to fix this.
